kernel/arch/dreamcast/sound/snd_mem.c

```diff
@@ -137,7 +148,7 @@ void snd_mem_shutdown(void) {
 /* Allocate a chunk of SPU RAM; we will return an offset into SPU RAM. */
 uint32 snd_mem_malloc(size_t size) {
     snd_block_t *e, *best = NULL;
-    size_t best_size = 4 * 1024 * 1024;
+    size_t best_size = 8 * 1024 * 1024;
```

Comment:
So I went to read the code to understand why you were doing this. This change is OK, but the new value makes it look like it actually means something - while in practice it's just an impossibly high value for a block size, so I'd advise to use `SIZE_MAX`, just so that it's clear.